depthai_helpers/calibration_utils.py

```diff
@@ -536,44 +453,37 @@ def calibrate_stereo(self, allCorners_l, allIds_l, allCorners_r, allIds_r, imsiz
                 return -1, "Stereo Calib failed due to less common features"
 
         stereocalib_criteria = (cv2.TERM_CRITERIA_COUNT +
-                                cv2.TERM_CRITERIA_EPS, 50000, 1e-9)
+                                cv2.TERM_CRITERIA_EPS, 1000, 1e-9)
```

Comment:
lets keep it high. It might take longer. but would prefer a longer optimization than okish calibration